controllers/sriovoperatorconfig_controller.go

```diff
@@ -91,17 +92,19 @@ func (r *SriovOperatorConfigReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 		return reconcile.Result{}, nil
 	}
 
-	// Render and sync webhook objects
-	if err = r.syncWebhookObjs(defaultConfig); err != nil {
-		return reconcile.Result{}, err
-	}
+	if enableAdmissionController {
+		// Render and sync webhook objects
+		if err = r.syncWebhookObjs(defaultConfig); err != nil {
+			return reconcile.Result{}, err
+		}
 
-	// Render and sync CA configmap
-	if err = r.syncCAConfigMap(types.NamespacedName{Name: INJECTOR_SERVICE_CA_CONFIGMAP, Namespace: req.Namespace}); err != nil {
-		return reconcile.Result{}, err
-	}
-	if err = r.syncCAConfigMap(types.NamespacedName{Name: WEBHOOK_SERVICE_CA_CONFIGMAP, Namespace: req.Namespace}); err != nil {
-		return reconcile.Result{}, err
+		// Render and sync CA configmap
+		if err = r.syncCAConfigMap(types.NamespacedName{Name: INJECTOR_SERVICE_CA_CONFIGMAP, Namespace: req.Namespace}); err != nil {
```

Comment:
Can we move the `enableAdmissionController` checking inside `syncCAConfigMap`? I think later we might want to add webhook support in vanilla k8s.